src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/magicnumber/InputMagicNumberWithBitwiseOperatorsWithIgnoreFieldsFalse.java

```diff
@@ -0,0 +1,57 @@
+/*
+MagicNumber
+ignoreNumbers = (default)-1, 0, 1, 2
+ignoreHashCodeMethod = (default)false
+ignoreAnnotation = (default)false
+ignoreFieldDeclaration = true
+ignoreAnnotationElementDefaults = (default)true
+constantWaiverParentToken = (default)TYPECAST, METHOD_CALL, EXPR, ARRAY_INIT, UNARY_MINUS, \
```

Comment:
We changed default, but we didn't change all other  Inputs with "(default)".
Please do.
I am not sure in what issue we will do validation of default, I will investigate later on.

Reply:
https://github.com/checkstyle/checkstyle/issues/15983 is created to improve separately.

Reply:
@mahfouz72 , please update other Inputs with default on this property to have such tokens.

Reply:
This will be a huge update we have many inputs for this check :)

Reply:
We need it, I hope it is just copy paste

Reply:
Done. It was painful because I had to change row numbers in every unit test

Reply:
Hm, we always keep 2 empty lines in config comment to avoid line shift.

Reply:
The addition is 2 lines, and I can't remove the 2 empty lines in the config comment because there will be no empty lines at all and we have a test for this. 

It will be good if we had 3 empty lines not 2

Reply:
Ok, next time try to reuse empty line in header comment, if post.